api/image.go

```diff
@@ -31,10 +31,12 @@ var ImageUploadedEvent = func(imageID, uploadPath, filename string) *event.Image
 }
 
 // ImagePublishedEvent returns an ImagePublished event for the provided path
-var ImagePublishedEvent = func(path string) *event.ImagePublished {
+var ImagePublishedEvent = func(filepath, filename, imageID, variant string) *event.ImagePublished {
```

Comment:
This feels like a constructor method - should this belong on the `event` package?  E.g `event.NewImagePublished(...)`

Reply:
This method exists purely for unit testing whereby the function is overridden by a mocked function that returns an invalid event. I'm not a fan of this implementation either but it was existing working code I didn't want to interfere with as part of this task. I simply changed the argument name as it was clashing with an imported package. There is a similar method used for the ImageUploadedEvent above it and I'd want to do the same for both if I were to change it.